### DISCUSSION TITLE
Google Api migration for AngularSocialLoign@ and Angular 10

### DISCUSSION THREAD

**Discussion by DamianRuczynski** [abacritt/angularx-social-login#727] [Q&A] [answered]
Hello all,
Quick question: Is Angularx-Social-Login (version 3) still supported for Angular 10? Also, does it handle migrating from the Google Login API's JavaScript implementation?
Thanks in advance for any info!

**Comment by Heatmanofurioso** [comment, marked as answer]
Hi @DamianRuczynski 

It does not support those cases

**Reply by DamianRuczynski** [reply]
ok, thank you
